Rakefile

```diff
@@ -59,8 +59,12 @@ namespace :arm do
         md = REGEN_METADATA[dir.to_sym]
         ar_base_command = "#{OS.windows? ? '' : 'mono '} #{REGEN_METADATA[:autorest_loc]}"
 
-        command = "#{ar_base_command} -i #{md[:spec_uri]} -pv #{md[:version]} -n #{md[:ns]} -pn #{md[:pn].nil? ? dir : md[:pn]} -g Azure.Ruby -o lib"
-        command += " -m #{md[:modeler]}" unless md[:modeler].nil?
+        if md[:spec_uri].end_with?('.json')
```

Comment:
Already confirmed. Once the flip has happened and everything is fine, it will be removed eventually. So, no issues as of now.